src/schema/v2/artwork/__tests__/artwork.test.js

```diff
@@ -2297,6 +2299,45 @@ describe("Artwork type", () => {
         expect(data.artwork.hasCertificateOfAuthenticity).toBe(true)
       })
     })
+
+    it("is set to proper object when certificate_of_authenticity and coa_from_authenticating_body are true", () => {
+      artwork.certificate_of_authenticity = true
+      artwork.coa_by_authenticating_body = true
+      return runQuery(query, context).then((data) => {
+        expect(data.artwork.certificateOfAuthenticity).toEqual({
+          label: "Certificate of authenticity",
+          details: "Included (issued by authorized authenticating body)",
+        })
+        expect(data.artwork.hasCertificateOfAuthenticity).toBe(true)
+      })
+    })
+
+    it("is set to proper when object certificate_of_authenticity and coa_from_gallery are true", () => {
```

Comment:
#superminor: probably should be `to proper object when`....

Reply:
thanks for catching that and taking a look. fixed! just waiting on final A-OK from Arielle and then will merge!